README.md

```diff
@@ -1,2 +1,63 @@
-# hello-world-project
-Hello World Project - Day 2 - Week 01
+<a name="readme-top"></a>
+
+
+<div align="center">
+  <img src="logo.png" alt="logo" width="140" height="auto" />
+  <br/>
+  <h3><b>Microverse First Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+# 📖 Hello World Microverse <a name="about-project"></a>
+
+**Hello World Microverse** is our first project. We were practicing all related about: Git/GitHub, Linters, using Templates and a few of HTML/CSS.
+
+## 🛠 Built With Visual Studio Code and NPM<a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+We built this project using tools as: Git client, Visual Studio Code and NPM.
+
+### Key Features <a name="key-features"></a>
+
+We developed this exercise to test some skills using Git/GitHub and Linters.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This job was suggested only for educative purposes.
+
```

Comment:
Kindly make sure that you have all the main headings present in your README file from [this template](https://github.com/microverseinc/readme-template) provided by Microverse. The only optional section is FAQ. You should delete the live demo section if you decide not to deploy the application. Make sure you have added the section **Future Features**, including 1-3 potential features to be added. Also, make sure to remove the default Microverse logo that is present in the provided template. Optionally you can use the logo which you are using currently.

Reply:
Thanks for your detailed feedback, my friend Muneeb @muneebulrehman. I'm working on it right now because I need to finish this step without errors to do my next step. 

Kind regards, man 

Javier.-

Reply:
Ok, my friend Muneeb @muneebulrehman I think that I followed your suggestions. When you have time, my man, I will be really happy if you check my updates. Thanks in advance.

Javier.-

Reply:
@jlcarrascof , kindly submit for a re-review if you are done.

Reply:
@muneebulrehman I just did it, man ... My apologies but I'm still learning about Github.